src/components/NcProgressBar/NcProgressBar.vue

```diff
@@ -113,6 +113,7 @@ export default {
 	// Browser specific rules
 	&::-webkit-progress-bar {
 		height: var(--progress-bar-height);
+		background-color: var(--color-background-dark)
```

Comment:
Or

```suggestion
		background-color: transparent
```

Reply:
Also need the `;` at the end of the line.

Reply:
done